File: finah-desktop/src/finah_desktop/Vraag.java
package finah_desktop;

public class Vraag {
	private int id;
	private String vraagstelling;
	private Foto afbeelding;
	private Geluid geluidsfragment;

	public Vraag() {

	}

	public Vraag(String vraagstelling, Foto afbeelding, Geluid geluidsfragment) {
		this.vraagstelling = vraagstelling;
		this.afbeelding = afbeelding;
		this.geluidsfragment = geluidsfragment;
	}

	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	public String getVraagstelling() {
		return vraagstelling;
	}

	public void setVraagstelling(String vraagstelling) {
		this.vraagstelling = vraagstelling;
	}

	public Foto getAfbeelding() {
		return afbeelding;
	}

	public void setAfbeelding(Foto afbeelding) {
		this.afbeelding = afbeelding;
	}

	public Geluid getGeluidsfragment() {
		return geluidsfragment;
	}

	public void setGeluidsfragment(Geluid geluidsfragment) {
		this.geluidsfragment = geluidsfragment;
	}
}

File: finah-web/PHP/Models/Vraag.php
<?php

    /**
     * Created by PhpStorm.
     * User: Gert
     * Date: 24/03/2015
     * Time: 15:57
     */
    class Vraag
    {
        public $Id;
        public $Vraagstelling;
        public $Afbeelding;
        public $Geluid;
        public $Thema;


        public function __construct($id = null, $Vraagstelling = null, $Afbeelding = null, $Geluid = null, $Thema = null)
        {
            $this->Id = $id;
            $this->Vraagstelling = $Vraagstelling;
            $this->Afbeelding = $Afbeelding;
            $this->Geluid = $Geluid;
            $this->Thema = $Thema;
        }

        public function getId()
        {
            return $this->Id;
        }

        public function setId($Id)
        {
            $this->Id = $Id;
        }

        public function getVraagstelling()
        {
            return $this->Vraagstelling;
        }

        public function setVraagstelling($Vraagstelling)
        {
            $this->Vraagstelling = $Vraagstelling;
        }

        public function getAfbeelding()
        {
            return $this->Afbeelding;
        }

        public function setAfbeelding($Afbeelding)
        {
            $this->Afbeelding = $Afbeelding;
        }

        public function getGeluid()
        {
            return $this->Geluid;
        }

        public function setGeluid($Geluid)
        {
            $this->Geluid = $Geluid;
        }

        public function getThema()
        {
            return $this->Thema;
        }

        public function setThema($Thema)
        {
            $this->Thema = $Thema;
        }
    }